### DISCUSSION TITLE
Welcome to Learning-JAVA Discussions!

### DISCUSSION THREAD

**Discussion by krishna09-dev** [krishna09-dev/Learning-JAVA#1] [Announcements]
<h1><b>HOWDY 😁</b></h1>
<h3>Hey i am keeping all my codes which i am making during learing java. It will be greater help if u look out through it and leave some suggestion which could help me to grow and learn new  things</h3>
<h2>I hope i get few suggestion through u guys...😊</h2>

**Comment by TariqAlhathloul** [comment]
I'm going to take a look👍

**Reply by krishna09-dev** [reply]
thanks a lot...

**Comment by krishna09-dev** [comment]
Thanks a lot.
Please let me know if you have some advice. It would be helpful for me...
